src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -1,509 +1,104 @@
-        wbsElementId: updated.wbsElementId,
-        detail: buildChangeDetail(
-          'Start Date',
-          wbsElement.workPackage?.startDate.toLocaleDateString() || 'null',
-          actCr.startDate.toLocaleDateString()
-        )
-      });
+  static async createActivationChangeRequest(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { body } = req;
+      const id = ChangeRequestService.createActivationChangeRequest(
+        body.submitterId,
+        body.wbsNum.carNumber,
+        body.wbsNum.projectNumber,
+        body.wbsNum.workPackageNumber,
+        body.type,
+        body.projectLeadId,
+        body.projectManagerId,
+        body.startDate,
+        body.confirmDetails
+      );
+      return res.status(200).json({ message: `Successfully created activation change request with id #${id}` });
+    } catch (error: unknown) {
+      next(error);
     }
-
-    changes.push({
-      changeRequestId: updated.crId,
-      implementerId: reviewerId,
-      wbsElementId: updated.wbsElementId,
-      detail: buildChangeDetail('status', wbsElement.status, WBS_Element_Status.ACTIVE)
-    });
-
-    await prisma.change.createMany({ data: changes });
-    await prisma.wBS_Element.update({
-      where: { wbsElementId: updated.wbsElementId },
-      data: {
-        projectLeadId: actCr.projectLeadId,
-        projectManagerId: actCr.projectManagerId,
-        workPackage: { update: { startDate: actCr.startDate } },
-        status: WBS_Element_Status.ACTIVE
-      }
-    });
-  }
-  // find userSettings that submitted the change request
-  const personSettings = await prisma.user_Settings.findUnique({ where: { userId: updated.submitterId } });
-  if (personSettings && personSettings.slackId) {
-    await sendSlackCRReviewedNotification(personSettings.slackId, updated.crId);
-  }
-
-  // TODO: handle errors
-  return res.status(200).json({ message: `Change request #${updated.crId} successfully reviewed.` });
-};
-
-export const createActivationChangeRequest = async (req: Request, res: Response) => {
-  const { body } = req;
-
-  // verify user is allowed to create activation change requests
-  const user = await prisma.user.findUnique({ where: { userId: body.submitterId } });
-  if (!user) {
-    return res.status(404).json({ message: `user with id #${body.submitterId} not found` });
   }
-  if (user.role === Role.GUEST) return res.status(403).json({ message: 'Access Denied' });
-
-  // verify wbs element exists
-  const wbsElement = await prisma.wBS_Element.findUnique({
-    where: {
-      wbsNumber: {
-        carNumber: body.wbsNum.carNumber,
-        projectNumber: body.wbsNum.projectNumber,
-        workPackageNumber: body.wbsNum.workPackageNumber
-      }
+  static async createStageGateChangeRequest(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { body } = req;
+      const id = ChangeRequestService.createStageGateChangeRequest(
+        body.submitterId,
```

Comment:
this should be removed and replaced with a call to `getCurrentUser` from the utils file and same with any other function that is using the id of the user sending the request (see editRisk)